CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Use `null` instead of "unitless` ([#55](https://github.com/stactools-packages/modis/pull/55))
```

Comment:
Hm, I don't remember if it was considered best practice or not. My instinct was to keep it, because an explicit `null` is more information than an omission (an omission could be "I forgot to add it").